### ISSUE TITLE
Support for collections with large numer of documents

### ISSUE THREAD

**Issue by luisgarcc** [nlplab/brat#1143]
I have a large document collection of ~50K+ documents. The documents themselves are small, about the one or two sentences, but the documents don't render. I keep getting the following error: 

    Error: ActiongetCollectionInformation failed on error Gateway Time-out

I have turned off validation but still no luck.

Other than partitioning the collection, are there any other suggestions to be able to view documents in the collection? What are the current known limits of document collection sizes?

**Comment by mefarazath** [comment]
Yeah i had to face some difficulty on the other extreme of this problem. I had a document of about 75k words(Sinhalese) and brat was damn slow understandably. I partitioned the document to about 20 small ones (each of about 4000 words) and brat handles it pretty well.

**Comment by pozz82** [comment]
Hi, I get the same error also with large (thousands of entries) annotation.conf
Please, consider also this together with large documents...

**Comment by acadTags** [comment]
Just get the same problem with over 10k documents and over 400 entities in some documents.
